#===================================================================
#RCS file: /cvsroot/mozilla/gfx/src nsRenderingContextImpl.h,v
#retrieving revision 1.23
#retrieving revision 1.24
diff -u -r1.23 -r1.24
--- mozilla/gfx/src/nsRenderingContextImpl.h 2007/03/27 06:21:551.23
+++ mozilla/gfx/src/nsRenderingContextImpl.h 2007/03/27 08:53:081.24
@@ -317,9 +317,8 @@
 protected:
   nsPenMode   mPenMode;
 private:
-  static nsIDrawingSurface*  gBackbuffer;         //singleton backbuffer 
-    // Largest requested offscreen size if larger than a full screen.
-  static nsSize            gLargestRequestedSize;
+  // Largest requested offscreen size if larger than a full screen.
+  static nsSize gLargestRequestedSize;
 
 };
 
